OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -31,14 +31,10 @@ public class WithInfantryBodyInfo : UpgradableTraitInfo, IRenderActorPreviewSpri
 
 		public IEnumerable<IActorPreview> RenderPreviewSprites(ActorPreviewInitializer init, RenderSpritesInfo rs, string image, int facings, PaletteReference p)
 		{
-			var facing = 0;
 			var ifacing = init.Actor.TraitInfoOrDefault<IFacingInfo>();
-			if (ifacing != null)
-				facing = init.Contains<FacingInit>() ? init.Get<FacingInit, int>() : ifacing.GetInitialFacing();
-
-			var anim = new Animation(init.World, image, () => facing);
+			var anim = new Animation(init.World, image, () => ifacing != null ? init.Contains<FacingInit>() ? init.Get<FacingInit, int>() : ifacing.GetInitialFacing() : 0);
```

Comment:
The `init` and `GetInitialFacing` aren't going to change, so moving them here hurts perf without giving us anything.  Can we keep this as it was?